modules/core/src/main/scala/shop/Main.scala

```diff
@@ -21,7 +21,7 @@ object Main extends IOApp {
             clients <- HttpClients.make[IO](cfg.paymentConfig, res.client)
             programs <- Programs.make[IO](cfg.checkoutConfig, algebras, clients)
             api <- HttpApi.make[IO](algebras, programs, security)
-            _ <- BlazeServerBuilder[IO]
+            _ <- BlazeServerBuilder[IO](scala.concurrent.ExecutionContext.global: scala.concurrent.ExecutionContext)
```

Comment:
I think it is more common to do `import scala.concurrent.ExecutionContext` at the top and then use it as `ExecutionContext.global`.